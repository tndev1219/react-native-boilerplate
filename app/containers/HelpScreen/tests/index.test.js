import React from 'react';
import { render } from '@testing-library/react';
import { IntlProvider } from 'react-intl';

import HelpScreen from '../index';

describe('<HelpScreen />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <HelpScreen />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
