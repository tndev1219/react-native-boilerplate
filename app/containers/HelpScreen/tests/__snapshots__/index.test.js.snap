// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelpScreen /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the HelpScreen container!
  </span>
</h1>
`;
